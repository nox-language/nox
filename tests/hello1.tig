(
    print("Hello, World!\n");
    print("2\n");
    printi(42)
)
