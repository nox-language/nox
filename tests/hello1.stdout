Hello, World!
2
42
